lib/restaurant.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'order'
+require_relative 'menu'
+
+class Restaurant
+  attr_reader :order
+
+  def initialize(menu, messenger = nil)
+    @menu = menu
+    @menu.read_menu
```

Comment:
Maybe have `Menu` call `read_menu` on itself, rather than relying on `Restaurant` to do it.  This will improve the encapsulation of `Menu`